webapp/views/partials/home.jade

```diff
@@ -0,0 +1,8 @@
+div(class='container')
+    h1 {{userName}}
+
+    div(class='div-center')
+        button(type='button', class='btn-circle', style='margin-bottom: 30px; margin-top: 50px;' ng-click='match()') Find a Match
```

Comment:
I think a general rule of thumb is to not put style information in HTML. It might be good to give the button a name (tag?) and providing styling for that name in the css file.